docs/userguide.md

```diff
@@ -0,0 +1,151 @@
+# User Guide
+
+## What is the cluster registry?
+
+The cluster registry is a Kubernetes-style API and API server that provides an
+endpoint for interacting with a list of clusters and associated metadata. If it
+helps, you can think of the cluster registry as a hosted kubeconfig file.
+However, since it's a Kubernetes API server, the cluster registry allows custom
```

Comment:
Aren't all these the properties of a kubernetes-style API, not the API server?

Reply:
That's a good point. Changed.